src/main/java/frc/robot/turret/TurretSubsystem.java

```diff
@@ -7,25 +7,34 @@
 
 import java.util.Objects;
 
+import static frc.robot.turret.TurretConstants.LOGGING_PATH;
+
 public class TurretSubsystem extends GBSubsystem {
 
-    ITurret turret;
-    TurretState state;
-    TurretInputsAutoLogged inputs;
+    private ITurret turret;
+    private TurretState state;
+    private TurretInputsAutoLogged inputs;
+    private Translation2d targetPoint;
 
     public TurretSubsystem(ITurret turret) {
         this.turret = turret;
+
         inputs = new TurretInputsAutoLogged();
         turret.updateInputs(inputs);
 
         this.state = TurretState.REST;
+        setTargetPoint(TurretConstants.LOOKING_TARGET);
```

Comment:
I might pass the target you want to the constructor (or overload it to set a default), personally I'm not a fan of class constructors setting values explicitly. Not a must but consider this.

Reply:
i think i might add a state for each target (like AMP and SPEAKER)

Reply:
and not get any target